aggregator/rollup_broadcaster.go

```diff
@@ -164,3 +336,26 @@ func (b *RollupBroadcaster) BroadcastOperatorSetUpdate(ctx context.Context, mess
 func (b *RollupBroadcaster) GetErrorChan() <-chan error {
 	return b.errorChan
 }
+
+func (b *RollupBroadcaster) tryGetOperatorSetUpdateById(ctx context.Context, avsReader chainio.AvsReaderer, operatorSetUpdateId uint64) ([]opsetupdatereg.RollupOperatorsOperator, error) {
```

Comment:
Need to rename this one to `tryGetOperatorSetUpdateById`

Reply:
Done in [cf14930](https://github.com/NethermindEth/near-sffl/pull/68/commits/cf14930a988b41877ac3c6e7a45d8310554a60fe)